### PR TITLE
AG-8997 Polar chart axis hole

### PR DESCRIPTION
**Pull request by alexanderby**

### CODE REVIEW

File: packages/ag-charts-community/src/chart/polarChart.ts [hunk 1]
```diff
@@ -40,25 +40,31 @@ export class PolarChart extends Chart {
...
-            }
+        const angleAxis = this.axes.find((axis) => axis.direction === ChartAxisDirection.X);
+        const radiusAxis = this.axes.find((axis) => axis.direction === ChartAxisDirection.Y);
+        if (!(angleAxis instanceof PolarAxis) || !(radiusAxis instanceof PolarAxis)) {
+            return;
+        }
+
+        const angleScale = angleAxis.scale;
+        const angles = angleScale.ticks?.().map((value: string) => angleScale.convert(value));
+        const innerRadiusOffset = radiusAxis.innerRadiusOffset;
```
Comment (manapeirov) [inline packages/ag-charts-community/src/chart/polarChart.ts:51]: Why do we need consts for this? can't we just do
`angleAxis.innerRadiusOffset = radiusAxis.innerRadiusOffset;`

**Review by manapeirov — APPROVED** [review]
LGTM

File: packages/ag-charts-community/src/chart/polarChart.ts (hunk 1 above)
Comment (alexanderby) [reply packages/ag-charts-community/src/chart/polarChart.ts:51]: The variable is reused in several places and makes the code look shorter.

Comment (manapeirov) [reply packages/ag-charts-community/src/chart/polarChart.ts:51]: Sure it does

**Pull request merged**